test/test_font_load.c

```diff
@@ -4,153 +4,139 @@
 #include <LCUI/gui/css_library.h>
 #include <LCUI/gui/css_parser.h>
 #include "test.h"
+#include "libtest.h"
 
 #define GetSegoeUIFont(S, W) LCUIFont_GetId( "Segoe UI", S, W )
 #define GetArialFont(S, W) LCUIFont_GetId( "Arial", S, W )
 
-int test_segoe_ui_font_load( void )
+void test_segoe_ui_font_load( void )
 {
-	int ret = 0, id;
 	LCUI_Font font;
+	int id;
 	if( LCUIFont_LoadFile( "C:/windows/fonts/segoeui.ttf" ) != 0 ) {
-		return 0;
+		return;
 	}
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeui.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeuib.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeuii.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeuil.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeuisl.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/segoeuiz.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/seguibl.ttf" ) == 0 );
-	CHECK( LCUIFont_LoadFile( "C:/windows/fonts/seguili.ttf" ) == 0 );
-	CHECK_WITH_TEXT(
-		"check get font (style: normal, weight: normal)",
-		id = GetSegoeUIFont( FONT_STYLE_NORMAL, FONT_WEIGHT_NORMAL )
-	);
+	it_i("Load segoeui.ttf", LCUIFont_LoadFile( "C:/windows/fonts/segoeui.ttf" ), 0 );
```

Comment:
Please use `clang-format` to format the code

Reply:
I've used clang-format on all affected c files.